src/test/java/core/basesyntax/FruitStorageTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorageTest {
+    Map<String, Integer> expected = new HashMap<>();
+    CsvFileReader reader = new CsvFileReader();
+    FruitStorage storage = new FruitStorage();
+
+    @Before
+    public void Before() {
+        expected.put("banana", 97);
+        expected.put("apple", 300);
+    }
```

Comment:
if the idea of this to expand enlarge quantity every time you call the next test - it's okay. But if it's not - move it to BeforeClass method

Reply:
fixed